app/components/elements/FontStyles.js

```diff
@@ -205,9 +205,10 @@ export const TopNavLink = Styled(NavLink)`
 
 export const TextLink = Styled(Link)`
   color: ${colors.text.link};
-  text-decoration: none;
-  line-height: ${lineHeights[0]};
+  font-size: ${fontSizes[1]}px;
   font-family: ${fonts.default};
+  line-height: ${lineHeights[0]};
```

Comment:
As with font size, perhaps we should let the line height be determined by the parent context